pkgs/by-name/ri/rigel-engine/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  git,
+  SDL2,
+  SDL2_mixer,
+  buildOpenGLES ? false,
+}:
+
+stdenv.mkDerivation {
+  pname = "rigel-engine";
+  version = "0-unstable-2024-05-26";
+
+  src = fetchFromGitHub {
+    owner = "lethal-guitar";
+    repo = "RigelEngine";
+    rev = "f05996f9b3ad3b3ea5bb818e49e7977636746343";
+    hash = "sha256-iZ+eYZxnVqHo4vLeZdoV7TO3fWivCfbAf4F57/fU7aY=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    git
```

Comment:
in the meantime i build the package on my computer. It seems like the `git` dependency is not necessary. can you please remove it?